content/en/_index.html

```diff
@@ -63,85 +65,49 @@ <h2 class="home-section-title home-section-title-projects">Projects</h2>
+            <p>A high-performance NIO network library, dedicated to building reliable, efficient services with a focus on RPC scenarios.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/volo">
             <h2>Volo</h2>
-            <p>A high-performance and strong-extensibility Rust RPC framework.</p>
+            <p>High Performance RPC Framework for high-performance & extensibility, in Rust.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
-        <a href="https://github.com/bytedance/monoio">
-            <h2>Monoio</h2>
-            <p>A thread-per-core Rust runtime with io_uring/epoll/kqueue.</p>
+        <a href="https://github.com/cloudwego/cwgo">
+            <h2>Cwgo</h2>
+            <p>All-in-one code Generation for efficient development.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/motore">
             <h2>Motore</h2>
-            <p>Async middleware abstraction powered by AFIT and RPITIT.</p>
+            <p>Empowering extensibility with async middleware abstraction, powered by AFIT and RPITIT.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/pilota">
             <h2>Pilota</h2>
-            <p>A thrift and protobuf implementation in pure rust with high performance and extensibility.</p>
-        </a>
-    </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
-        <a href="https://github.com/bytedance/sonic">
-            <h2>Sonic</h2>
-            <p>A blazingly fast JSON serializing & deserializing library, accelerated by JIT and SIMD.</p>
-        </a>
-    </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
-        <a href="https://github.com/cloudwego/frugal">
-            <h2>Frugal</h2>
-            <p>A very fast dynamic Thrift serializer & deserializer based on just-in-time compilation.</p>
-        </a>
-    </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
-        <a href="https://github.com/cloudwego/fastpb">
-            <h2>Fastpb</h2>
-            <p>A faster Protobuf serializer & deserializer.</p>
-        </a>
-    </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
-        <a href="https://github.com/cloudwego/cwgo">
-            <h2>Cwgo</h2>
-            <p>CloudWeGo All in one Code Generator.</p>
-        </a>
-    </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
-        <a href="https://github.com/cloudwego/shmipc-go">
-            <h2>Shmipc</h2>
-            <p>A high-performance IPC solution based on shared memory IPC.</p>
+            <p>Protobuf and Thrift implementation in pure Rust for high performance & extensibility.</p>
```

Comment:
I prefer to leave Monoio and Sonic in the homepage, those are core projects.

Reply:
Which other 2 projects can we remove to put back Monoio and Sonic? @GuangmingLuo

Reply:
leaving Kitex, Hertz, Volo, Netpoll, Monoio, Sonic, cwgo